From e314c200c12a27d279a47356686e170bf449996d Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 3 Feb 2012 12:26:40 -0800
Subject: [PATCH 1/3] Revert "ASoC: Mark WM5100 register map cache only when
 going into BIAS_OFF"

This reverts commit 11a17e56ac9c607c6eaecf64b618cd17c828ade0
(e53e417331c57b9b97e3f8be870214a02c99265c upstream) as it breaks the
build.

Reported-by: Tim Gardner <rtg.canonical@gmail.com>
Cc: Mark Brown <broonie@opensource.wolfsonmicro.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 sound/soc/codecs/wm5100.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/codecs/wm5100.c b/sound/soc/codecs/wm5100.c
index d0beeec..c745ce8 100644
--- a/sound/soc/codecs/wm5100.c
+++ b/sound/soc/codecs/wm5100.c
@@ -1406,7 +1406,6 @@ static int wm5100_set_bias_level(struct snd_soc_codec *codec,
 		break;
 
 	case SND_SOC_BIAS_OFF:
-		regcache_cache_only(wm5100->regmap, true);
 		if (wm5100->pdata.ldo_ena)
 			gpio_set_value_cansleep(wm5100->pdata.ldo_ena, 0);
 		regulator_bulk_disable(ARRAY_SIZE(wm5100->core_supplies),
-- 
1.7.9.4

